DigitalLearningSolutions.Web/Helpers/CustomPolicies.cs

```diff
@@ -7,26 +7,44 @@ public class CustomPolicies
         public const string UserOnly = "UserOnly";
         public const string UserCentreAdminOnly = "UserCentreAdminOnly";
         public const string UserFrameworksAdminOnly = "UserFrameworksAdminOnly";
+        public const string UserCentreManagerOrUserUserAdminOnly = "UserCentreManagerOrUserUserAdminOnly";

```

Comment:
Could we name this "UserCentreManager" or something similar as "UserUserAdmin" is a super admin that should always see everything and I think we're OK to exclude it from the names.